docs/community_resources/translations.md

```diff
@@ -5,6 +5,54 @@ If you'd like to see eZ Platform in your language, you can contribute to the tra
 [`ezplatform-i18n`](https://github.com/ezsystems/ezplatform-i18n) contains the XLIFF files providing translations.
 You can use an XLIFF editor of your choice to contribute strings in your language.
 
+## How to translate the interface using Crowdin
+
+If you wish to contribute to an existing translation of Back Office or start a new one, the best way is to start with in-context translation (but you can also [translate directly on the Crowdin website](#translating-outside-in-context)).
+
+## Preparing to use in-context
+
+To start translating, you need an option to turn in-context translation on and off. To do this, set a browser cookie. There are several ways to do this, we will highlight a couple here.
```

Comment:
To make it more accurate: "To enable the in-context translation UI, a pseudo-language locale needs to be used in HTTP requests. To make this easier, a cookie `ez_in_context_translation=1` can be set. There are several ways to do this. We will highlight a couple down below."